change-validation/src/test/scala/bootstrap/rudder/plugin/MigrateSupervisedGroupsTests.scala

```diff
@@ -0,0 +1,82 @@
+package bootstrap.rudder.plugin
+
+import better.files.File
+import com.normation.JsonSpecMatcher
+import com.normation.plugins.changevalidation.MockSupervisedTargets
+import com.normation.rudder.rest.RestTestSetUp
+import com.normation.zio.UnsafeRun
+import java.nio.file.Files
+import org.junit.runner.RunWith
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+import org.specs2.specification.AfterAll
+
+@RunWith(classOf[JUnitRunner])
+class MigrateSupervisedGroupsTests extends Specification with JsonSpecMatcher with AfterAll {
+  isolated
```

Comment:
it needs to be just `sequential`, not `isolated`, else the whole set up is done again, which is costly and can lead to error depending of the millesecond of directory creation, for ex: 

![image](https://github.com/Normation/rudder-plugins/assets/44649/49a60069-0f00-4058-bb8f-59756163fbeb)

Reply:
(previously I got a "directory already exist", which was more explicit)